src/assets/toolkit/styles/components/thumbnail.css

```diff
@@ -51,7 +51,7 @@
   transition: border 300ms ease; /* 6 */
 }
 
-.Thumbnail:matches(:focus, :hover)::after {
+.Thumbnail:matches(:--enter)::after {
```

Comment:
Is `:matches` necessary in every case? I notice [it's missing from cssnext's example](http://cssnext.io/features/#custom-selectors).